app/components/layout/cards/tools_component.rb

```diff
@@ -0,0 +1,12 @@
+module Layout
+  module Cards
+    class ToolsComponent < ApplicationComponent
```

Comment:
Nitpick: "Tools" seems like an unusual name for the component and the section. The text isn't really about tools we offer? I know is says "Data-driven tools" in the heading above, but I don't think what we're listing are tools, or that the component needs to be limited to that.

Could we use something more descriptive. E.g. a StatementComponent? Or BadgedStatementComponent? Or ProductFeatureComponent?

Reply:
Makes sense, thanks

Reply:
Have renamed StatementComponent to SectionHeader (which made more sense anyway), so I could then rename ToolsComponent to StatementComponent. I've renamed the tools yaml section to features